examples/simple.yml

```diff
@@ -148,17 +148,23 @@ data-silos:
             description: The first name of the user, inputted during onboarding
             categories:
               - category: USER_PROFILE
+                name: Name
```

Comment:
This would fail right? bc we don't seed any sub categories/purposes with names other than "Other"...

Reply:
Yes it's for in the future when they do have their own defined subcategories.